eagles-food-backend/appsettings.json

```diff
@@ -1,6 +1,7 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "To be changed to prod db"
+    "DefaultConnection": "To be changed to prod db",
+    "TestConnection": "Server=MYSQL8002.site4now.net;Database=db_a9ef58_teagle;Uid=a9ef58_teagle;Pwd=c7b9@r.jDR7.ZKW"
```

Comment:
also put your specific settings in the Development one, itll override the others on your machine, and is ignored by git (i think)

Reply:
Alright would do